### PR TITLE
Display inbound proof library mapping

### PR DESCRIPTION
**Pull request by michalsidzej**
Resolves L2B-2585

### CODE REVIEW

File: packages/frontend/src/view/components/BlockchainAddress.tsx [hunk 1]
```diff
@@ -48,28 +50,39 @@ export function BlockchainAddress(props: Props) {
...
+            </Tooltip>
+          </span>
           {addressInfo.name.length > 0 && (
-            <span className="whitespace-nowrap text-xs text-zinc-500">
-              ({addressInfo.name})
-            </span>
+            <>
+              {' '}
+              <span className="whitespace-nowrap text-xs text-zinc-500">
+                {addressInfo.name}
```
Comment (sdlyy) [inline packages/frontend/src/view/components/BlockchainAddress.tsx:72]: Do we want to remove brackets around the verified contract name?

Comment (sdlyy) [reply packages/frontend/src/view/components/BlockchainAddress.tsx:72]: image

**Review by sdlyy — APPROVED** [review]
LG, take a look at bracketing of an verified address please

File: packages/frontend/src/view/components/BlockchainAddress.tsx (hunk 1 above)
Comment (michalsidzej) [reply packages/frontend/src/view/components/BlockchainAddress.tsx]: [4a13884](https://github.com/l2beat/lz-monitoring/pull/97/commits/4a138840cda40c5eba275be585ddc8debd6e959a)

**Pull request merged**